src/components/Thanks/ThanksPageSingleVersion.vue

```diff
@@ -1,3 +1,37 @@
 <template>
-	<p>Content goes here</p>
+	<div class="tw-bg-eco-green-1 tw-p-3 md:tw-py-4 tw-flex tw-flex-col tw-gap-2.5">
+		<div class="content-box tw-mx-auto">
+			<div
+				v-if="!isGuest"
+			>
+				<LoanComment
```

Comment:
Does our old TY also show commenting for each loan in the checkout?

Reply:
Not really, we have a selectedLoan, but probably worth a try to ask product for consistency with a case for multiple loans?

Reply:
Sounds good, thanks for the info.